kotlin/build.gradle

```diff
@@ -32,10 +31,10 @@ dependencies {
     implementation "io.github.cdimascio:dotenv-kotlin:6.2.2"
     implementation "org.ini4j:ini4j:0.5.4"
 
-    implementation "io.ktor:ktor-client:$ktorVersion"
-    implementation "io.ktor:ktor-client-okhttp:$ktorVersion"
-    implementation "io.ktor:ktor-client-json:$ktorVersion"
-    implementation "io.ktor:ktor-client-gson:$ktorVersion"
```

Comment:
Removed for now but can add back if we decide to have a low-level ktor client to make the requests